src/dotnet/Notification.Service/Db/DbMessage.cs

```diff
@@ -0,0 +1,33 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace ActualChat.Notification.Db;
+
+[Table("Messages")]
+[Index(nameof(DeviceId))]
+public class DbMessage : IHasId<string>
```

Comment:
1) I would rename it to `DbNotification`
2) Am I right `Id` includes `DeviceId` (otherwise how we shard it?)